.github/workflows/releasecandidate.yml

```diff
@@ -68,9 +68,34 @@ jobs:
           channel: '#team-devops'
           name: 'Workflow Status'
 
+  extract-rc-version:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.extract-rc-version.outputs.version }}
+      rc_number: ${{ steps.extract-rc-version.outputs.rc_number }}
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
+      - name: Extract version and RC number
+        id: extract-rc-version
+        run: |
+          TAG_NAME=${GITHUB_REF#refs/tags/}
+          if [[ $TAG_NAME =~ ^v([0-9]+\.[0-9]+\.[0-9]+)-rc([0-9]+)$ ]]; then
```

Comment:
Regex assumes tag names will strictly be of the form `v[0-9].[0-9].[0-9]-rc[0-9]` i.e. no other characters in the string

Reply:
possibly, this is okay though